src/test/java/dataloader/CsvLoaderTest.java

```diff
@@ -56,8 +56,9 @@ public void testValidCsvFileForMap() throws IOException {
 
 
     /**
-     * This is the same test as above
-     * but for type Table.
+     * Test Scenario: a csv file is provided
```

Comment:
Perhaps ```a data set in CSV format is provided...``` would have been better here or not?

Reply:
did the update.